contrib/cron-jobs/update-asn-data

```diff
@@ -1 +0,0 @@
-../../intelmq/bots/experts/asn_lookup/update-asn-data
```

Comment:
please leave it as is for backwards-compatibility

Reply:
Done.